airflow/operators/python.py

```diff
@@ -489,7 +489,9 @@ def __init__(  # pylint: disable=too-many-arguments
         self.pickling_library = dill if self.use_dill else pickle
 
     def execute(self, context: Dict):
-        serializable_context = {key: context[key] for key in self._get_serializable_context_keys()}
+        serializable_context = {
+            key: context[key] for key in self._get_serializable_context_keys() if context.get(key) is not None
+        }
```

Comment:
Can you explain this change?

Reply:
With the previous code, if I passed an empty context object to test the execute method, it would throw a KeyError when it tries to do `key: context[key]`. Essentially, if I can't provide a context object with all the keys from _get_serializable_context_keys, an KeyError will be thrown. 

Most operator execute method tests I have seen use an empty context object as param to test. I tried to do the same here which required the change above.

Reply:
@cccs-seb have you tried using `mock.MagicMock()` as context? In this way we probably won't need this change and we can do even more - for example assert that some values from the context were accessed. WDYT?

Reply:
Great idea - I'll make the change as soon as I can.  Thanks for the suggestion!